builders/rust/builder.toml

```diff
@@ -0,0 +1,12 @@
+[[buildpacks]]
+id = "dev.boson.rust"
+uri = "docker://quay.io/boson/faas-rust-bp:{{VERSION}}"
+
+[[order]]
+[[order.group]]
+id = "dev.boson.rust"
+
+[stack]
+id = "dev.boson.stacks.rust"
+build-image = "quay.io/boson/faas-stack-build:rust-{{VERSION}}"
+run-image = "quay.io/boson/faas-stack-run:rust-{{VERSION}}"
```

Comment:
Now that we have multiple buildpacks that result in a binary for runtime execution, we should probably have a single tag for these vs.  `boson/faas-stack-run:rust-{{VERSION}}` and `boson/faas-stack-run:go-{{VERSION}}` which are bit for bit the same image outside of the tag.

Perhaps this should point to `quay.io/boson/faas-stack-run:ubi8-minimal-{{VERSION}}`. It has all of the necessary metadata, and we can avoid adding another tag to the image.